src/core/viz/expressions/viewportAggregation.js

```diff
@@ -2,19 +2,244 @@ import BaseExpression from './base';
 import { number } from '../functions';
 import * as schema from '../../schema';
 
+/**
+ * Return the average value of the features showed in the viewport
+ *
+ * @param {carto.expressions.property} property - Column of the table
+ * @return {carto.expressions.number} Result of the aggregation
+ *
+ * @example
+ * const s = carto.expressions;
+ * const $amount = s.prop('amount');
+ * const viz = new carto.Viz({
+ *   @v_avg: s.viewportAvg($amount)
```

Comment:
I wouldn't use an alias, can be confusing to users

Reply:
I'm using variables here because using this in a style expression would add more extra complexity. And this is a legit use of the API, because maybe the user wants to show the numeric value in a Widget so it's valid